layers/+distribution/spacemacs-base/packages.el

```diff
@@ -300,7 +300,7 @@ Example: (evil-map visual \"<\" \"<gv\")"
         i.e. `SPC m h h`, to lookup the source of the definition,
         while falling back to `evil-lookup'"
         (interactive)
-        (condition-case nil
+        (condition-case-unless-debug nil
```

Comment:
Even if this function is written to catch an exception it shouldn't catch
everything just the one it anticipates.
On Thu, Feb 25, 2016 at 3:43 PM Eivind Fonn <notifications@github.com>
wrote:

> In layers/+distribution/spacemacs-base/packages.el
> <https://github.com/syl20bnr/spacemacs/pull/5217#discussion_r54161030>:
>
> > @@ -300,7 +300,7 @@ Example: (evil-map visual \"<\" \"<gv\")"
> >          i.e. `SPC m h h`, to lookup the source of the definition,
> >          while falling back to `evil-lookup'"
> >          (interactive)
> > -        (condition-case nil
> > +        (condition-case-unless-debug nil
>
> I agree [image: :+1:]
>
> My perspective is that you enable debugging because you want a backtrace
> and I'd just assume always generate one.
>
> Coming from a Python perspective, errors are frequently used as part of an
> interface. That seems to be the case here, even though as you say it should
> have been written differently to begin with.
>
> IOW, *errors* are not always *exceptions*.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/syl20bnr/spacemacs/pull/5217/files#r54161030>.
>